package.json

```diff
@@ -1,17 +1,40 @@
 {
-    "name": "",
-    "version": "",
-    "description": "",
-    "author": "",
-    "license": "",
-    "engines": {
-      "node": ">=6.13.0"
-    },
-    "scripts": {
-    },
-    "dependencies": {
-    },
-    "devDependencies": {
-      "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
-    }
-  }
\ No newline at end of file
+  "name": "reviews",
+  "version": "",
+  "description": "Front end capstone project",
+  "author": "Janice Lam",
+  "license": "ISC",
+  "engines": {
+    "node": ">=6.13.0"
+  },
+  "scripts": {
+    "react-dev": "webpack -d --watch",
+    "test": "jest"
+  },
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "express": "^4.15.0",
+    "jquery": "^3.1.1",
```

Comment:
@janicelam21 

As a general note, using jquery and react together is generally a no-no (if this was just for AJAX methods, feel free to disregard, but I might recommend axios or fetch!).

Reply:
The jquery was just going to be for AJAX requests, but I can just use axios. I'll remove jquery.